client/components/library/BooksTable.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table } from 'react-materialize';
+
+import BookRow from './BookRow';
+
+
+/*
+
+ */
+const BooksTable = (
+  { bookList, tableHeaders, borrowBook }
+) => {
+  const books = bookList ? bookList.map(book =>
+    (
+      <BookRow
+        key={book.id}
+        {...book}
+        onButtonClick={borrowBook}
+      />
+    )
+  ) : null;
+  const headers = tableHeaders.map(header =>
+    <th key={header}>{header}</th>
+  );
+  return (
+    <Table
+      className="striped centered bordered"
+    >
+      <thead>
+        <tr>
+          {headers}
+        </tr>
+      </thead>
+      <tbody>
+        {books}
```

Comment:
Your books defaults to null if the bookList props passes a 'Falsy' value down to the BooksTable component. Don't you think a feedback message would be better for your user experience ??